src/Umbraco.Web.BackOffice/Controllers/TourController.cs

```diff
@@ -21,21 +26,41 @@ public class TourController : UmbracoAuthorizedJsonController
     private readonly IContentTypeService _contentTypeService;
     private readonly TourFilterCollection _filters;
     private readonly IHostingEnvironment _hostingEnvironment;
+    private readonly IWebHostEnvironment _webHostEnvironment;
```

Comment:
Looks like `IWebHostEnvironment` has replaced `IHostingEnvironment` in this controller. Can we remove the `IHostingEnvironment` field and the same entry in the new constructor?

Reply:
I think the problem with that was that the controller will have the exact same number of params so the DI container won't know which controller to prefer. The `[ActivatorUtilitiesConstructor]` annotation didn't work in this case.

Reply:
@elit0451 interesting - `ActivatorUtilitiesConstructor` ShouldJustWork™️ 🤔 well... if the DI gets confused, maybe leave the `IHostingEnvironment` in the constructor with a comment why it's there, but still remove the unused field?